opm/simulators/wells/WellState.cpp

```diff
@@ -825,8 +825,8 @@ WellState::parallelWellInfo(std::size_t well_index) const
     return ws.parallel_info;
 }
 
-template void WellState::updateGlobalIsGrup<ParallelWellInfo::Communication>(const ParallelWellInfo::Communication& comm);
-template void WellState::communicateGroupRates<ParallelWellInfo::Communication>(const ParallelWellInfo::Communication& comm);
+template void WellState::updateGlobalIsGrup<Parallel::Communication>(const Parallel::Communication& comm);
+template void WellState::communicateGroupRates<Parallel::Communication>(const Parallel::Communication& comm);
```

Comment:
This appears to rely on `ParallelCommunication.hpp` being brought in through transitive includes.  I'd recommend adding the header directly.

Reply:
done